doc/sg/c_cpp_coding_style.md

```diff
@@ -414,7 +414,7 @@ __attribute__((section(".crt"))) void _crt(void);
 ### Nonstandard Compiler Builtins
 
 In order to avoid a total reliance on one single compiler, any nonstandard compiler builtins (also known as intrinsics) should be used via a single canonical definition.
```

Comment:
nit: **In order to avoid total reliance on a single compiler**

Reply:
Fixed